roles/openshift_openstack/tasks/unprovision.yml

```diff
@@ -1,4 +1,25 @@
 ---
+- name: Get kuryr services
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+  oc_obj:
```

Comment:
One more thing: this will fail if we only run provision but not install (because the masters won't have the `oc` binary or anything else installed).

I'd still like the uninstall to work in that case (as it does now) -- maybe you provision to test your DNS or packages or something, find some issues and want to tear it down.

Can we add `ignore_errors: true` here?